packages/lesswrong/server/voteServer.ts

```diff
@@ -71,6 +74,12 @@ const addVoteServer = async ({ document, collection, voteType, extendedVote, use
     },
     {}
   );
+  if (userHasElasticsearch(null) && collectionIsAlgoliaIndexed(collection.collectionName)) {
```

Comment:
Do we want to sync a document each time it's voted on in perpetuity, or is it just part of the initial setup phase?